src/components/SetupDesktop/LocalNetworkAccessPrompt.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "preact/hooks";
+import {
+  checkLocalNetworkAccessPermission,
+  getChromeVersion,
+  LocalNetworkAccessState,
+} from "../../utils/localNetworkAccess";
+import styles from "./LocalNetworkAccessPrompt.module.pcss";
+
+export function LocalNetworkAccessPrompt() {
+  const [permissionState, setPermissionState] = useState<LocalNetworkAccessState>("unsupported");
+  const [chromeVersion, setChromeVersion] = useState<number | null>(null);
+
+  useEffect(() => {
+    const version = getChromeVersion();
+    setChromeVersion(version);
+
+    if (version && version >= 142) {
+      checkLocalNetworkAccessPermission().then(setPermissionState);
+    }
```

Comment:
nit - This is already covered within `checkLocalNetworkAccessPermission`, so can keep this simpler

(And remove the chrome version state and part of the code below)

```suggestion
    checkLocalNetworkAccessPermission().then(setPermissionState);
```